conf/modules/collective_tracking_control_target.xml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="collective_tracking_control_target" dir="ctc">
+  <doc>
+	  <description>Distributed algorithm for tracking a velocity/position reference signal. Module only for the target to broadcast its position and velocity to the pursuers.
```

Comment:
Same comment that the other xml file

Reply:
Sure, I agree. I will do that for the next PR.